app/views/simulator/edit.html.erb

```diff
@@ -214,6 +214,8 @@
       <div id="combinationalAnalysis" style="display:none;" title="BooleanLogicTable"></div>
       <div id="insertSubcircuitDialog" style="display:none;" title="Insert SubCircuit"></div>
       <div id="openProjectDialog" style="display:none;" title="Open Project"></div>
+      <div id="saveonline" style="display:none;" title="Project Name"></div>
+      <div id="newcircuitcreate" style="display:none;" title="Create New Circuit"></div>
```

Comment:
I know the initial design had separate div for each dialog, but is it possible to have a common div for each?

Reply:
I guess there can be 1 div inside that these div's be present

Reply:
or like circuit elements we can create div